### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by Sp3EdeR** [mach64/trakt-scrobbler#303] [General]
Hello Kroot,

It seems to me that the master version of the script is working quite well now (definitely better than 1.6.3). I wonder if you're planning a new release version any time soon?

**Comment by iamkroot** [comment]
Hi, yes. I just finished trying it out on Windows yesterday. I am waiting for some more folks to try out the master version. Will probably release a beta version today. Thanks for providing feedback!
